src/platform/Kconfig

```diff
@@ -37,6 +37,13 @@ config PANTHERLAKE
 	help
 	  Select if your target platform is Pantherlake-compatible
 
+config WILDCATLAKE
+	bool "Build for Wildcatlake"
+	select ACE
+	select ACE_VERSION_3_0
```

Comment:
select `ACE_VERSION_3_0` but other files use `CONFIG_SOC_INTEL_ACE30`. Is it correct?

Reply:
I checked, and it looks like the `ACE_VERSION_x_0` definitions set the value to `ACE_VERSION`. This is mainly used by the dai driver. I agree that this needs to be changed in a separate pr.